src/List/ListItem.js

```diff
@@ -38,6 +39,16 @@ export const styleSheet = createStyleSheet('ListItem', (theme) => {
       paddingLeft: 16,
       paddingRight: 16,
     },
+    button: {
```

Comment:
Stolen from `Button`, but not sure if correct approach. But `<ListItem button>` didnt have proper hover effect of `Button`